OCCommunicationLib/OCCommunicationLibTests/OCCommunicationLibTests.m

```diff
@@ -921,7 +921,7 @@ -(void)testReadFile{
     //4.- Get and Compare the etag of the same folder with the preview, if is different the TEST is OK
     
     //Block Store Attributes
-    __block long long etag = 0;
+    __block long long etag = @"";
```

Comment:
Wrong assignation.
Until we do not merge the code from Develop the test will not work on Travis. Anyway I will fix this on master.
Thanks @DeepDiver1975